src/supermarket/app/controllers/api/v1/cookbook_versions_controller.rb

```diff
@@ -76,4 +93,9 @@ def require_evaluation_params
     params.require(:cookbook_version)
     params.require(:foodcritic_failure)
   end
+
+  def require_collaborator_params
+    params.require(:cookbook_name)
+    params.require(:collaborator_failure)
+  end
```

Comment:
Maybe add `params.require(:collaborator_message)` because in this context, we don't have a case where we don't have a message to display. (Unlike foodcritic, where there may be no findings for the message because it passed.)